### DISCUSSION TITLE
Abnormal HTML tags appear below the article title

### DISCUSSION THREAD

**Discussion by goldsheep3** [nunocoracao/blowfish#1036] [Q&A] [answered]
*I'm sorry, I'm from China, and my English is not very good, so I used ChatGPT to help with the translation of this content. Please understand.*

I'm trying to deploy a website to Github Pages, but during this process, some unexpected issues occurred: the time, word count, and reading time positions have HTML tags.

<img width="640" alt="e0ec41d90c221915f76a531f401ce623" src="https://github.com/nunocoracao/blowfish/assets/80096970/991e4830-3726-4eca-9357-e1011ec7a039">

*(Please see the latest post; the image loading issue is due to the network and is a normal occurrence locally.)*

The results generated using `hugo server` locally are normal.

<img width="640" alt="4cb6d2a0ddf55d5a121642e32a721aa4" src="https://github.com/nunocoracao/blowfish/assets/80096970/61568b3b-6ace-4bb3-a492-acbf3e73241a">

I would like to know how to solve this issue. My repository is located at `goldsheep3/star-with-land`. Thanks.

**Comment by myseq** [comment, marked as answer]
Seems like an error with Hugo. Are you installing Hugo with snap ? You have to revert back to Hugo 0.119 or refresh your Blowfish with "git submodule update --remote --merge"

**Reply by goldsheep3** [reply]
Thank you!
I installed Hugo using scoop in a Windows 11 environment.
I tried running git submodule update --remote --merge in the root directory, and after successfully running it and redeploying, the issue did not occur again.
Thanks again!
